main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/LuisMG96/academy-go-q42021/server"
+	"log"
```

Comment:
For best practices, you can separate the core packages, your packages and third packages like following:

Go core

Project Local 

Third-Party